src/coverlet.console/Program.cs

```diff
@@ -128,10 +136,14 @@ static int Main(string[] args)
                       doesNotReturnAttributesValue,
                       excludeAssembliesWithoutSourcesValue,
                       sourceMappingFileValue);
-        context.ExitCode = taskStatus;
+        //context.ExitCode = taskStatus;
```

Comment:
Could we leave this in and then just 
```csharp
return await config.InvokeAsync(args).ConfigureAwait(false);
```

after we changed the main method to 
```csharp
static async Task<int> Main(string[] args)
```

If that alternative would work we could also get rid of the `Environment.Exit(...)`

Reply:
I replaced `Environment.Exit(s_exitCode);` with `return s_exitCode;`.